src/hooks/useOAuthFlow.tsx

```diff
@@ -57,6 +61,15 @@ export const OAuthContextProvider = ({
     authConfig.usePKCE = true;
   }
 
+  if (inviteId && evc) {
+    authConfig.additionalParameters = {
```

Comment:
Does this make it so we can accept an invite and login at the same time?

Reply:
Not quite - it facilitates a "magic link" user signup, though.  Then the invite acceptance afterwards still happens from the InviteProvider in this PR.